README.md

```diff
@@ -9,4 +9,4 @@ in Julia.
 
 **Documentation:** [![Latest](https://img.shields.io/badge/docs-latest-blue.svg)](https://juliastats.github.io/StatsModels.jl/latest)
 
-Originally part of [DataFrames.jl](http://github.com/JuliaStats/DataFrames.jl).
+Originally part of [DataTables.jl](http://github.com/JuliaStats/DataTables.jl).
```

Comment:
Yes. I would wait until things have stabilized a bit more, and then explain how the package is supposed to be used. Anyway we need generic interfaces very soon to support DataFrames.